spynl/tests/conftest.py

```diff
@@ -20,11 +20,10 @@ def settings():
     return {'spynl.pretty': '1',
             'enable_plugins': [],
             'pyramid.debug_authorization': 'false',
-            'pyramid.default_locale_name': 'en',
+            'pyramid.default_locale_name': 'nl',
             'pyramid.reload_templates': 'true',
             'spynl.domain': 'localhost',
             'spynl.languages': 'en,nl',
-            'default_locale_name': 'nl',
```

Comment:
Note: the "default_locale_name" setting doesnt exist but the "pyramid.default_locale_name" exists